repositories/src/main/scala/cromwell/pipeline/datastorage/dto/FileParameter.scala

```diff
@@ -0,0 +1,43 @@
+package cromwell.pipeline.datastorage.dto
+
+import org.mongodb.scala.Document
+import play.api.libs.json._
+
+final case class FileParameter(name: String, typedValue: TypedValue)
+
+object FileParameter {
+  import OptionFormat._
+
+  implicit object InputNodeFormat extends Format[FileParameter] {
+    override def writes(o: FileParameter): JsValue = {
+      val (valType, value) = TypedValue.toPair(o.typedValue)
+      JsObject(
+        Seq(
+          "name" -> JsString(o.name),
+          "type" -> JsString(valType),
+          "value" -> (value match {
+            case Some(value) => JsString(value)
+            case None        => JsNull
+          })
+        )
+      )
+    }
+
+    override def reads(json: JsValue): JsResult[FileParameter] = {
+      val typeVal =
+        TypedValue.fromPair(
+          (json \ "type").as[String],
+          (json \ "value").as[Option[String]]
+        )
+      JsSuccess(FileParameter((json \ "name").as[String], typeVal))
+    }
+  }
+
+  def toDocument(node: FileParameter): Document = Document(Json.stringify(Json.toJson(node)))
+
+  def fromDocument(document: Document): FileParameter =
```

Comment:
`toDocument` and `fromDocument` are unused methods.

Reply:
Yes, I deleted them, thx